requests_toolbelt/adapters/appengine.py

```diff
@@ -29,8 +29,28 @@
        >>> appengine.monkeypatch()
 
 which will ensure all requests.Session objects use AppEngineAdapter properly.
+
+If you should need to disable certificate validation when monkeypatching (to
+force third-party libraries that use Requests to not validate certificates, if
+they do not provide API surface to do so, for example), you can disable it:
+
+   .. code-block:: python
+
+       >>> from requests_toolbelt.adapters import appengine
+       >>> appengine.monkeypatch(validate_certificate=False)
+
+   .. warning::
+
+       If ``validate_certificate`` is ``False``, the monkeypatched adapter
+       will *not* validate certificates. This effectively sets the
+       ``validate_certificate`` argument to urlfetch.Fetch() to ``False``. You
+       should avoid using this wherever possible. Details can be found in the
+       `documentation for urlfetch.Fetch()`__.
+
+       __ https://cloud.google.com/appengine/docs/python/refdocs/google.appengine.api.urlfetch  # noqa
```

Comment:
Same thing as above, although this looks duplicated from there. Perhaps remove the duplication and add a reference to it, e.g.,

```rst
You are also able to :ref:`disable certificate validation <insecure_appengine>` when monkey-patching.
```